crates/turbopack-node/src/pool.rs

```diff
@@ -359,15 +373,30 @@ impl NodeJsOperation {
 
         Ok(status)
     }
+
+    pub fn kill_on_drop(&mut self) {
+        self.kill_on_drop = true;
+    }
 }
 
 impl Drop for NodeJsOperation {
     fn drop(&mut self) {
-        if let Some(process) = self.process.take() {
-            self.processes
-                .lock()
-                .unwrap()
-                .push(NodeJsPoolProcess::Running(process));
+        if let Some(mut process) = self.process.take() {
+            if self.kill_on_drop {
```

Comment:
this is way worse
just add `cmd.kill_on_drop(true);` in `NodeJsPoolProcess::new`, we always want that, no?

this is a feature of tokio

Reply:
```rs
            if !self.in_bad_state {
                self.processes
                    .lock()
                    .unwrap()
                    .push(NodeJsPoolProcess::Running(process));
            }
```
would be more obvious

Reply:
ah I see